src/ray/util/tests/shared_lru_test.cc

```diff
@@ -23,6 +23,19 @@ namespace ray::utils::container {
 
 namespace {
 constexpr size_t kTestCacheSz = 1;
+
+class TestClass {
```

Comment:
We can have a more descriptive name like `StringLike` or `StringWithAbslHash`

Reply:
See my comments below (https://github.com/ray-project/ray/pull/48954/files#r1865347940), to implement heterogeneous access, you need to implement more comparison methods.

Reply:
I know, but TestClass is not a good name for that. Maybe `EqAndHashable` ?

Reply:
OK, updated.